TESTS/mbed_drivers/lp_timeout/main.cpp

```diff
@@ -119,29 +121,48 @@ void lp_timeout_500us(void)
 
 }
 
-utest::v1::status_t greentea_failure_handler(const Case *const source, const failure_t reason) {
+utest::v1::status_t greentea_failure_handler(const Case * const source, const failure_t reason)
+{
     greentea_case_failure_abort_handler(source, reason);
     return STATUS_CONTINUE;
 }
 
 Case cases[] = {
+    Case("Callback called once (with attach)", test_single_call<AttachTester<LowPowerTimeout> >),
+    Case("Callback called once (with attach_us)", test_single_call<AttachUSTester<LowPowerTimeout> >),
+
+    Case("Callback not called when cancelled (with attach)", test_cancel<AttachUSTester<LowPowerTimeout> >),
+    Case("Callback not called when cancelled (with attach_us)", test_cancel<AttachUSTester<LowPowerTimeout> >),
+
+    Case("Callback override (with attach)", test_override<AttachTester<LowPowerTimeout> >),
+    Case("Callback override (with attach_us)", test_override<AttachUSTester<LowPowerTimeout> >),
+
+    Case("Multiple timeouts running in parallel (with attach)", test_multiple<AttachTester<LowPowerTimeout> >),
+    Case("Multiple timeouts running in parallel (with attach_us)", test_multiple<AttachUSTester<LowPowerTimeout> >),
```

Comment:
These two tests with multiple `LowPowerTimeout` objects fail on _K64F_, _NUCLEO_F070RB_, _NUCLEO_L476RG_. Waiting for the outcome of #5150.